src/hotspot/share/gc/g1/g1YoungCollector.cpp

```diff
@@ -1083,7 +1099,7 @@ void G1YoungCollector::collect() {
     // modifies it to the next state.
     jtm.report_pause_type(collector_state()->young_gc_pause_type(_concurrent_operation_is_full_mark));
 
-    policy()->record_young_collection_end(_concurrent_operation_is_full_mark, evacuation_failed());
+    policy()->record_young_collection_end(_concurrent_operation_is_full_mark, evacuation_alloc_failed());
```

Comment:
The arg name (where this method is defined) should be updated to sth like `evac_alloc_failed` from `evacuation_failure`.